pyanaconda/modules/network/installation.py

```diff
@@ -180,6 +180,10 @@ def _write_interface_rename_config(self, root, ifname_option_values, overwrite):
         :type overwrite: bool
         """
 
+        # Avoid link files for 'nbft' interfaces as it breaks Boot from NVM over TCP
+        ifname_option_values = [value for value in ifname_option_values
+                                if not value.startswith('nbft')]
```

Comment:
why not use the `is_nbft_device` function?

Reply:
Updated